README.md

```diff
@@ -1,2 +1,168 @@
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|title|LocalizedText|e.g. "Beta Blockers for Heart Failure"|May be localized.|
+|url|LocalizedText|e.g. "https://youtu.be/XfgcXkq61k0"|May be localized.|
+
+Q: Can we specify a certain order in Firestore or should we add something like an id to order it by?
+Q: All videos are shown to all users, or is there any filtering/reordering/difference for any users with respect to other users?
+
+## /users/$userId$
+
+In this section, we describe all user-related data to be stored.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|fullName|string|-|Contains a user's full name. TBD: May potentially be split up into firstName/lastName or firstName/middleNames/familyName.|
+|dateOfBirth|Date|-|To be used for verification purposes.|
+|invitationCode|string|-|The invitationCode to be used when logging in to the app for the first time.|
+|language|optional string|e.g. "en"|Following IETF BCP-47 / [FHIR ValueSet languages](https://hl7.org/fhir/R4B/valueset-languages.html)|
+
+Q: Can we somehow ensure that a given logged in user can only access their own information? Clinicians require access to more patients, but they are currently not reflected as "users".
+
+### /users/$userId$/devices/$deviceId$
+
+This data is required to send push notifications over the [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging) System.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|modifiedDate|DateTime|-|This date is updated whenever the token is sent to the server, even if it is not replaced by a different token. It simply reflects the last date we can definitely confirm the token was active. Idea: We may ignore some devices, if the token has not been updated for a long time, since the app has not been opened for a long time.|
+|notificationToken|string|-|The FCM token as received from Firebase in the app. TBD: We may want to make this non-optional and simply not store device information, if there is no available token (e.g. if notifications are restricted).|
+|platform|optional string|e.g. "iOS", "Android"|This information is important as context for the `osVersion`, `appVersion` and `appBuild` properties.|
+|osVersion|optional string|e.g. "17.5.1"|The version of the OS. Depending on the OS, they may have different formats to be specified separately.|
+|appVersion|optional string|e.g. "1.0.1"|The version of the app as it is specified on the App/Play Store.|
+|appBuild|optional string|e.g. "56"|The build version of the app as defined by the CI pipeline releasing the app.|
+|language|optional string|e.g. "en"|Following IETF BCP-47 / [FHIR ValueSet languages](https://hl7.org/fhir/R4B/valueset-languages.html)|
+
+Q: Should we specify language/region by user or by device? Currently both a user and a device may contain a language. A notification shall prioritize a device's language setting, fall back to the user's language setting and use `en-US` in all other cases.
```

Comment:
I think the fallback idea is a good one.

Reply:
I rewrote it to not be a question.